src/R/Interpreters/Impl/IRInterpreterInfo.cs

```diff
@@ -33,6 +33,6 @@ public interface IRInterpreterInfo {
         /// </summary>
         /// <param name="fs"></param>
         /// <returns></returns>
-        bool VerifyInstallation(ISupportedRVersionRange svl = null, IFileSystem fs = null, ICoreShell coreShell = null);
+        bool VerifyInstallation(ISupportedRVersionRange svl = null, IFileSystem fs = null, IUIService ui = null);
```

Comment:
Yes, I'll fix some ICoreShell -> IServicesContainer and related where I find, but it will prob need another separate pass.